spec/jobs/bulk_update_job_spec.rb

```diff
@@ -25,6 +25,19 @@
       expect(r2.state).to eq ["complete"]
     end
 
+    context "one of the resources is taken down" do
+      let(:resource2) do
+        Timecop.freeze(Time.now.utc - 1.day) do
+          FactoryBot.create_for_repository(:scanned_resource, state: "takedown")
+        end
+      end
+      it "doesn't persist the one that was marked taken down" do
+        described_class.perform_now(ids: ids, args: args)
+        r2 = query_service.find_by(id: resource2.id)
+        expect(r2.updated_at.to_date).to be < Time.current.to_date
```

Comment:
Can you verify that the state is still takedown? When we add support for other fields to the bulk edit, we might want to update the rights statement, or add to a collection, but still leave the state alone.